src/theme/variables.css

```diff
@@ -396,4 +397,28 @@ hr {
   .tablet {
     display: unset;
   }
+
+  .header {
+    display: grid;
+    grid: "id       timeline" min-content
+          "cards    timeline" 1fr
+          / 1fr 500px;
+    justify-content: space-between;  
+  }
+
+  .id {
+    grid-area: id;
+  }
+  
+  .cards {
+    display: grid;
+    grid-template-columns: repeat(2, minmax(314px, max-content));
```

Comment:
we want to use auto repeat here, there can me more than just 2 columns.

Reply:
Ok